packages/cardano-services/src/cli.ts

```diff
@@ -166,7 +166,8 @@ withCommonOptions(
         ...args,
         postgresConnectionString: connectionStringFromArgs(args, ''),
         // Setting the projection names via env variable takes preference over command line argument
-        projectionNames: args.projectionNames ? args.projectionNames : projectionNames
+        projectionNames: args.projectionNames ? args.projectionNames : projectionNames,
+        ...handlePolicyIdsFromFile(args)
```

Comment:
It seems `handlePolicyIdsFromFile` is called here and also inside `loadProjector` function. Should it be called just once? Also since it returns a promise instead of modified args, using spread operator might have unexpected result.

Reply:
Yep, it's a refuse. Thank you

Reply:
2def8b682 should solve this